src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,474 @@
+﻿/* @internal */
+namespace ts.codefix {
+
+    type ImportCodeActionKind = "CodeChange" | "InsertingIntoExistingImport" | "NewImport";

```

Comment:
They are used to filter the code actions later. For example, inserting new item to an existing import list should take higher priority to adding a new import statement. In reality it is possible that we can get both kinds of code actions if the code has multiple existing imports